### PR TITLE
Some fixes and improvements

### PR DESCRIPTION
**Pull request by Sekku**
- Check the input to be an installed version
- Resolved issue #1

### CODE REVIEW

File: dotnet-sdk.cmd [hunk 1]
```diff
@@ -4,6 +4,15 @@ if [%1]==[help] goto help
...
 if [%1]==[latest] goto sdk_latest
+
+for /f %%f in ('dir /b "%programfiles%\dotnet\sdk"') do (
+    if %1==%%f goto swicth
+) 
+echo The %1 version of .Net Core SDK was not found 
+echo Please, run "dotnet sdk list" to make sure you have it installed in "%programfiles%\dotnet\sdk" 
+goto end
+
+:swicth
```
Comment (faniereynders) [inline dotnet-sdk.cmd]: Please fix typo: Swicth -> Switch

File: dotnet-sdk.cmd [hunk 2]
```diff
@@ -4,6 +4,15 @@ if [%1]==[help] goto help
 if [%1]==[] goto help
 if [%1]==[list] goto sdk_list
 if [%1]==[latest] goto sdk_latest
+
+for /f %%f in ('dir /b "%programfiles%\dotnet\sdk"') do (
+    if %1==%%f goto swicth
```
Comment (faniereynders) [inline dotnet-sdk.cmd]: Please fix typo: Swicth -> Switch

**Review by faniereynders — CHANGES_REQUESTED** [review]
Kindly fix some spelling errors, thanks!

**Comment by andrewlock** [comment]
Nice - though I wonder if the parent global.json check could be improved? This only checks the parent directory itself right? If you're going to warn about there being a global.json in a parent directly, you should probably walk all the way up the directory path. 

Given that you are explicitly creating a new global.json anyway, I think deleting parent ones is unnecessary, and has potentially undesirable consequences (deleting a global.json will affect other projects in sibling folders that were using it).

**Comment by Sekku** [comment]
For the first point, I made a mistake assuming it would be the _first_ parent folder the one that matters and not the higher ones, so yes, it should go up all the way but considering your second point (and that's why I made deleting the global.json in the parent folder optional), if it's there, it's for a reason

Maybe switching to the latest version could be changed to:
1. Check if there's a global.json in a parent folder
2. If there's one, ask whether the user wants to switch to the latest sdk installed or to version in the global.json
2.1 If the latest installed is selected => create a global.json in the current folder with it
2.2 If the global version is selected => Do nothing / delete current global.json
3. If there isn't any file, then just delete current global.json

**Comment by andrewlock** [comment]
Yup, that seems reasonable :) especially if we say what version is in the parent global.json!

**Comment by faniereynders** [comment]
Great work! Thank you!

**Pull request merged**